### DISCUSSION TITLE
使用 Hugo NexT.Gemini主题生产的静态网页不支持在二级目录访问

### DISCUSSION THREAD

**Discussion by babiwawa** [hugo-next/hugo-theme-next#115] [Q&A]
请按照此 Issue 模版提供相关信息，例如源码仓库、网站链接和屏幕截图，这将有助于我们进行调查。
(Please follow this Issue template to provide relevant information, such as source code repository, website URL and screenshots, which will help us investigate.)

## 问题自检列表 (Issue Checklist)

> 请确认如下情况已检查，并将 [ ] 换成 [x] 来选择 (Change [ ] to [x] to selected which your could make sure)

- [x] 已使用 Hugo Extended 0.86.0 或更高的版本 (I'm using Hugo Extended version 0.86.0 or later)
- [x] 已使用 Hugo NexT 4.0 或更高的版本 (I'm using Hugo NexT version 4.0 or later)
- [x] 已阅读 [Hugo 故障排除](https://gohugo.io/troubleshooting/)并尝试修复无果 (I had already read the [Troubleshooting page of Hugo](https://gohugo.io/troubleshooting/))
- [x] 已搜索过当前所有[已知问题](https://github.com/hugo-next/hugo-theme-next/issues)和[讨论区](https://github.com/hugo-next/hugo-theme-next/discussions/categories/q-a) ，但没有找到帮助 (I had already searched for current [Issues](https://github.com/hugo-next/hugo-theme-next/issues) and [Q & A](https://github.com/hugo-next/hugo-theme-next/discussions/categories/q-a), which does not help me.)

***

## 预期行为 (Expected behavior)

> 请描述你想实现的预期效果 (Please describe the expected behavior what you want)

## 实际行为 (Actual behavior)

> 请同时提供网站链接、屏幕截图和源码地址 (Please provide your site links, screenshots and code repository)

使用 Hugo NexT.Gemini主题生产的静态网页不支持在二级目录访问

![392}I$}N{PKQTUZ5D4FN %D](https://github.com/hugo-next/hugo-theme-next/assets/118897722/a276f389-abca-4d15-bd25-e1468ec99522)
生成的网站JS.css.img资源都无法呈现，点击网站文章和链接也都是错误的。
### 主题配置(Configuration)

> 提供主题 `config.yaml` 中修改过的部分内容 (Paste ONLY CHANGED CONFIGURATION from  `config.yaml`)

```yaml

```

## 其他有用信息 (Other Information)

> 例如：操作系统、浏览器等 (e.g. Browser, System)

**Comment by elkan1788** [comment]
这里看上去你的JS和CSS配置有问题，可以使用浏览器的F12开发者工具看看哪些文件出现404错误

**Comment by babiwawa** [comment]
![image](https://github.com/hugo-next/hugo-theme-next/assets/118897722/9bcf26ab-b50e-4672-bfba-d22e7e33f24c)

所有的CSS.JS.IMG文件都加载错误，资源应该是从http://localhost/blog/开始加载，但是实际情况是从http://localhost/这里开始加载的，所

以导致网站在二级目录出现错误，站点地址已经设置成二级目录了还是不行

![image](https://github.com/hugo-next/hugo-theme-next/assets/118897722/37b5d350-89ff-4c0c-a267-e0a73ef104d2)
